src/TorchSharp/NN/Sequential.cs

```diff
@@ -16,30 +16,32 @@ namespace Modules
         /// <summary>
         /// This class is used to represent a Sequential module.
         /// </summary>
-        public class Sequential : torch.nn.Module
+        public class Sequential : torch.nn.Module<Tensor, Tensor>
```

Comment:
As I mentioned in the discussion, it would make sense for a `Sequential<T>: Module<T, T>`